### ISSUE TITLE
AVT 3 - React Tabs Component, JAWS on Windows 10 has issues with reading tab content

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3473]
## Environment
macOS 10.14.5
Chrome 75.0.3770.100
VoiceOver

## Description
1. Tab to Tabs Component Tab Label 2 has focus and is read as expected by JAWS.
2. Arrow to Tab Labels and each tab is read as expected
3. Could not use any key sequence to read selected tab content.
**Expected Result:** Tab content should receive focus after the Tab Label 1 is selected, and Content for first tab goes here. should be announced by JAWS.
Fails: [WCAG 2.0 1.3.2 Meaningful Sequence](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

**Comment by dakahn** [comment]
@snidersd are you still seeing this error? I'll be on my Windows machine later this afternoon and can help confirm.

**Comment by snidersd** [comment]
@dakahn Retested and the issue is still present.

**Comment by asudoh** [comment]
@snidersd A question; Do you mean that JAWS doesn't have a way to read the tab content of moving reading cursor to tab content without setting regular keyboard focus (tab key or `.focus()`) to tab content? Ref: https://github.com/carbon-design-system/carbon/pull/4959#issuecomment-574392970

**Comment by snidersd** [comment]
@asudoh @dakahn JAWS does not read anything in the tabpanel. For example, testing the [ARIA Authoring Practice Tab example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html) with JAWS, I can use the following keys to read the tabs and tabpanel:
1. Tab to the example
2. Right and Left arrow keys navigate across the tabs
3. Insert+Alt+M reads the content of the tabpanel
I am unable to read the tabpanel using this key sequence or the JAWS reading and navigation keys.
@dakahn Were you able to try it on your Windows machine?

**Comment by dakahn** [comment]
Restested this with @snidersd and we seem to be able to reach the "controlled conten" via `INSERT+ALT+m`. So I'll close this down 👍
